### DISCUSSION TITLE
Compatible bikes?

### DISCUSSION THREAD

**Discussion by blancmunier** [cagnulein/qdomyos-zwift#390] [Q&A]
Hi again,
I was wondering if the compatibility with FTMS protocol was required to use the app?
For example, would it work with the cardiostrong BX70i bike work. This one has a controlled resistance and is working fine with iConsole+ app. Is there a chance that the controlled resistance would work with zwift then?
Thanks again !
Yvan

**Comment by cagnulein** [comment]
Yes I’m supporting control resistance also for the iconsole bikes

Il giorno gio 1 lug 2021 alle 23:11 blancmunier ***@***.***>
ha scritto:

> Hi again,
> I was wondering if the compatibility with FTMS protocol was required to
> use the app?
> For example, would it work with the cardiostrong BX70i bike work. This one
> has a controlled resistance and is working fine with iConsole+ app. Is
> there a chance that the controlled resistance would work with zwift then?
> Thanks again !
> Yvan
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cagnulein/qdomyos-zwift/discussions/390>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAALYWFPWYR724TQ5D5YMQ3TVTKXBANCNFSM47VPDVTQ>
> .
>
-- 
Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud

**Comment by misko903** [comment]
So it means every bike Which supports iconsole+ is compatible with your app?

**Comment by cagnulein** [comment]
Exactly!


Il giorno gio 18 gen 2024 alle 22:30 misko903 ***@***.***> ha
scritto:

**Reply by misko903** [reply]
Man, this is genial! Thanks!

**Reply by cagnulein** [reply]
thanks! remember to rate the app on store if you like, it means a lot to me! thanks!

**Reply by misko903** [reply]
I will, but I need to buy a spinner first. Looking for something which can be used with zwift and I came here.
